commands/done.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
```

Comment:
Since this code is repeated in each of your command files, it's a prime candidate for refactoring: create one function in a different module, and then require that module in each of your commands.

So say you created a file `readTasks.js`:

```js
function readTasks() {
  return JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
}

module.exports = readTasks
```

You could then use it in each of your `commands/*.js` files like this:

```js
const readTasks = require('../readTasks.js')

const tasks = readTasks()

// ...
```